src/modules/local_position_estimator/BlockLocalPositionEstimator.cpp

```diff
@@ -426,29 +515,31 @@ void BlockLocalPositionEstimator::updateHome()
 	_altHome = alt;
 	_gpsAltHome += delta_alt;
 	_baroAltHome +=  delta_alt;
-	_lidarAltHome +=  delta_alt;
-	_sonarAltHome +=  delta_alt;
+	_visionHome(2) += delta_alt;
```

Comment:
@jgoppert I don't think that having lidar and sonar with different offsets. If you mount both on the same base, lidar and sonar altitude differs by 2-3cm (at least if you use a liar like SF10a).